tc-package.sh

```diff
@@ -4,13 +4,9 @@ set -xe
 
 mkdir -p /tmp/artifacts
 
-tar -C ${HOME}/DeepSpeech/tf/bazel-bin/tensorflow/ \
```

Comment:
You wouldn't need to rebuild it all for code changes, just the files that change (and any dependents) - In terms of distribution, it would allow us to distribute libdeepspeech.so changes outside of libtensorflow.so, which would be nice. Static linking in a Bazel library build isn't supported though, so we'd either have to live with the gnuc99 warning or distribute libdeepspeech and libc_speech_features separately.

On the other hand, I do like that this is just one file to worry about installing and distributing, and I would suppose there's the possibility that there's a system libtensorflow.so that we might conflict with. Pros and cons :)